asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -418,10 +431,20 @@ def main():
 
     # Parse symlinks in output_dir
     output_dir = Path(args.output_dir)
+    # check that output_dir exists, otherwise create it
     if not output_dir.exists():
         output_dir.mkdir()
+
     logger = FileLogger("run_docking_oe", path=str(output_dir)).getLogger()
+    logger.info("Starting run_docking_oe")
+    logger.info(f"Output directory: {output_dir}")
+
+    if not output_dir.exists():
+        logger.info("Output directory does not exist, creating...")
+        output_dir.mkdir()
```

Comment:
this is redundant with lines 434-436, no?

Reply:
fixed